modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataRecordCollectionGetRequestExecutor.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.data.engine.internal.executor;
+
+import com.liferay.data.engine.service.DEDataRecordCollectionGetRequest;
+import com.liferay.data.engine.service.DEDataRecordCollectionGetResponse;
+import com.liferay.dynamic.data.lists.model.DDLRecordSet;
+import com.liferay.dynamic.data.lists.service.DDLRecordSetLocalService;
+
+/**
+ * @author Leonardo Barros
+ */
+public class DEDataRecordCollectionGetRequestExecutor {
+
+	public DEDataRecordCollectionGetRequestExecutor(
+		DDLRecordSetLocalService ddlRecordSetLocalService,
+		DEDataEngineRequestExecutor deDataEngineRequestExecutor) {
+
+		_ddlRecordSetLocalService = ddlRecordSetLocalService;
+		_deDataEngineRequestExecutor = deDataEngineRequestExecutor;
+	}
+
+	public DEDataRecordCollectionGetResponse execute(
+			DEDataRecordCollectionGetRequest deDataRecordCollectionGetRequest)
+		throws Exception {
+
+		long deDataRecordCollectionId =
```

Comment:
We dont need to declare this variable to be used only un the method above. We can call this method inside get record set.